docs/source/home.rst

```diff
@@ -2,9 +2,35 @@
 Home
 ####
 
-Vircadia is an open-source software where you can create and share virtual reality (VR) experiences. You can create and host your own VR world, explore other worlds, meet and connect with other users, attend or host live VR events and much more.
+Vircadia is open-source software which enables you to create and share virtual worlds as virtual reality (VR) and desktop experiences. You can create and host your own virtual world,
+explore other worlds, meet and connect with other users, attend or host live VR events, and much more.
+
+The Vircadia metaverse provides built-in social features, including avatar interactions, spatialized audio and interactive physics.
```

Comment:
```suggestion
The Vircadia metaverse provides built-in social features, including avatar interactions, spatialized audio, and interactive physics.
```

Reply:
My teachers would have given me shit for adding an *optional* comma
haha

Reply:
Heh, the reason for that is consistency. The rule is you don't have to do it, but if you do in one place then you've gotta do it everywhere ideally. :) I personally do it as well, to me it makes the English more clear, so that makes two of us.